docs/ReleaseHistory.md

```diff
@@ -194,7 +195,6 @@
 - BUG: Correct `AzureCosmosDBIdentifiableKey` rule id to `SEC101/160` (previously incorrectly listed as `SEC101/163`).
 - BUG: Correct length of `SEC101/166.AzureSearchIdentifiableQueryKey` and `SEC101/167.AzureSearchIdentifiableAdminKey` rules to 39 bytes and properly mark it as `DetectionMetadata.Identifiable`.
 - BUG: Remove `/AM7` signature + check from rust code.
-- NEW: Add `SEC101/190.AzureEventGridIdentifiableKey` check.
```

Comment:
>AzureEventGridIdentifiableKey [](http://example.com/codeflow?start=23&length=29)

This rule was *not* added in 1.4.24, it was added in 1.4.20 (and in fact referenced below).